code/__DEFINES/monitor.dm

```diff
@@ -0,0 +1,51 @@
+//How often we calculate the state
+#define START_STATE_CALCULATION 0 MINUTES
+#define FREQUENCY_STATE_CALCULATION 1 MINUTES
+
```

Comment:
is there a particular reason this needed a define?

Reply:
I always find it easier to have define in my code, it allows better readability